harmonica/tests/test_prism.py

```diff
@@ -52,6 +52,23 @@ def test_invalid_prisms():
         _check_prisms(np.atleast_2d([w, e, s, n, top, bottom]))
 
 
+def test_disable_checks():
+    "Check if the disable_checks flag works properly"
+    valid_prism = [0, 10, 0, 10, -10, 0]
+    invalid_prism = [0, 10, 0, 10, 0, -10]
+    density = 100
+    coordinates = [0, 0, 0]
+    # Check if an invalid prism doesn't raise an error with the disable_checks flag set to True
+    prism_gravity(
+        coordinates, invalid_prism, density, field="potential", disable_checks=True
+    )
+    # Check if an invalid prism returns the inverse of a valid one's return value
+    # with only top and bottom values flipped between the two.
+    assert prism_gravity(
+        coordinates, invalid_prism, density, field="potential", disable_checks=True
+    ) == (-1) * prism_gravity(coordinates, valid_prism, density, field="potential")
```

Comment:
This checks are great! I would simplify them in order to reduce repeated code and to avoid computing the same quantity twice: the gravity potential generated using the invalid prism is computed twice on lines 62 and 67.
One solution could be to store this result:
```python
    # Check if an invalid prism doesn't raise an error with the disable_checks flag set to True
    invalid_result = prism_gravity(
        coordinates, invalid_prism, density, field="potential", disable_checks=True
    )
    # Check if the invalid prism generates a potential field inverse
    # than the one generated by the valid prism.
    valid_result = prism_gravity(coordinates, valid_prism, density, field="potential")
    assert invalid_result == - valid_result
```